### DISCUSSION TITLE
Problem of importing ODF data with Import Wizard

### DISCUSSION THREAD

**Discussion by erdaye90** [mtex-toolbox/mtex#1431] [Ask Anything]
In the latest version of MTEX (5.8.1), I imported the ODF demo file which includes three euler angles and weight with Import Wizard and found that there was a format error. The Import Wizard told me that could not detect file format. However, when I use the old version, e.g., 5.1.1 and 5.3, there is no problem. I also found that when I start up mtex-5.8.1, the command showed a lot of warning, which could not happen in old version. The MATLAB I use now is R2016a. Is it possible that there is some conflicts between the old version MATLAB and the latest version MTEX? If not, why the demo ODF that MTEX provided could not be correctly imported by its Import Wizard?

**Comment by kilir** [comment]
Hi,
thank you for reporting that, it's a bug.
For the time being, you can for example use:
```
odf   = loadODF_generic('odf.txt','cs',crystalSymmetry('23'),'ColumnNames',{'Euler 1' 'Euler 2' 'Euler 3' 'weight'})
``` 
Cheers,
Rüdiger

**Reply by erdaye90** [reply]
Thanx! It works.
